static/js/message_lists.js

```diff
@@ -8,6 +8,14 @@ export function set_current(msg_list) {
     current = msg_list;
 }
 
+export function all_rendered_message_lists() {
+    const rendered_message_lists = [home];
+    if (current === message_list.narrowed) {
```

Comment:
I think it's safer to check `current !== home`, both because this module controls both of those things, and also because `message_list.narrowed` doesn't get reset when leaving a view, which is fine because `current` will have been reset, but I'd prefer to not be accessing stale state anywhere if we can avoid it.